examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainActivity.kt

```diff
@@ -1,13 +1,30 @@
 package com.revenuecat.purchasetester
 
+import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
+import com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI
+import com.revenuecat.purchases.Purchases
 import com.revenuecat.purchases_sample.R
 
 class MainActivity : AppCompatActivity() {
 
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        if (Purchases.isConfigured) {
+            Purchases.sharedInstance.handleDeepLink(intent)
+        }
     }
+
+    @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+    override fun onNewIntent(intent: Intent?) {
```

Comment:
Added this override since it might be needed if the activity is already there, like in this example